druid/src/data.rs

```diff
@@ -368,3 +368,29 @@ impl Data for piet::Color {
         self.as_rgba_u32().same(&other.as_rgba_u32())
     }
 }
+
+macro_rules! impl_Data {
```

Comment:
nit, but I would expect this to be all lowercase? Is there prior art for this sort of thing?

Reply:
I think prior art in this project is the `impl_value_type` macros in the Env module. The snake-case this has been updated to now matches that,